src/unfold/forms.py

```diff
@@ -92,41 +93,54 @@ def __init__(
 class UserChangeForm(BaseUserChangeForm):
     def __init__(
         self,
-        request: Optional[HttpRequest] = None,
         *args,
         **kwargs,
     ) -> None:
-        super().__init__(request, *args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
There is no "request" in this form. Only data as first argument. Why it worked fine: Djano uses it as a positional only -> the "request" was "data"